internal/hcs/schema2/virtual_pci_device.go

```diff
@@ -13,4 +13,5 @@ package hcsschema
 // docs when a public build with this is out.
 type VirtualPciDevice struct {
 	Functions []VirtualPciFunction `json:",omitempty"`
+	PropagateNumaAffinity bool	`json:"PropagateNumaAffinity,omitempty"`
```

Comment:
we should update the comment above. This particular field is `2.11` schema version and also pre-release/experimental.